modules/cadentApertureMXBidAdapter.js

```diff
@@ -390,6 +390,14 @@ export const spec = {
       if (uspConsent && typeof uspConsent.consentString === 'string') {
         consentParams.push(`usp=${uspConsent.consentString}`);
       }
+      if (gppConsent && typeof gppConsent === 'object') {
+        if (gppConsent.gppString && typeof gppConsent.gppString === 'string') {
+          consentParams.push(`gpp=${gppConsent.gppString}`);
+        }
+        if (gppConsent.applicableSections && typeof gppConsent.applicableSections === 'object') {
+          consentParams.push(`gpp_sid=${gppConsent.applicableSections}`);
+        }
```

Comment:
I believe these checks can be simplified since you already check that gppConsent is an object

```js
if (typeof gppConsent.gppString === 'string') {
```

```js
if (Array.isArray(gppConsent.applicableSections)) {
```

Or just use optional chaining

```js
if (typeof gppConsent?.gppString === 'string') {
```

```js
if (Array.isArray(gppConsent?.applicableSections)) {
```